app/common/utils.js

```diff
@@ -0,0 +1,7 @@
+export function shiftItemRandomly(array) {
+    return array.splice(getRandomInt(0, array.length), 1)[0];
+}
+
+export function getRandomInt(min, max) {
```

Comment:
Correct, although it is properly used since the non-inclusive upper bound is array's length. Nevertheless it is much clearer changing its name and including the upper bound because the usage is to randomly select an index of an array, and array.length tends to mean out of bound.